faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Reporting usage problems, security issues, and bugs
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red Error Datasets**:
+        - Refer to the [Troubleshooting errors](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting.html) FAQ for red error in datasets. 
+    - **Unexpected results in Green Success Dataset**:
+        - To resolve it you may be asked to send in a shared history link and possibly a shared workflow link. For sharing your history, refer to this [link](https://training.galaxyproject.org/training-material/faqs/galaxy/histories_sharing.html).
+        - To reach our support team, visit [Support FAQs](https://training.galaxyproject.org/training-material/faqs/galaxy/#support).
+    - **Functionality problems**:
+        - Using [Galaxy Help](https://help.galaxyproject.org/) is the best way to get help in most cases.
+        - If the problem is more complex, email a description of the problem and how to reproduce it. 
+    - **Administrative problems**:
+        - If the problem is present in your own Galaxy, the administrative configuration may be a factor.  
+        - For the fastest help directly from the development community, admin issues can be alternatively reported to the mailing list or the GalaxyProject [Gitter channel](https://gitter.im/galaxyproject/Lobby). 
+
+- For **Security Issues**, do not report them via GitHub. Kindly disclose these as explained in this [document](https://github.com/galaxyproject/galaxy/blob/dev/SECURITY_POLICY.md).
+- For **Bug Reporting**, create a Github issue. Include the steps mentioned [here](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting_reporting.html).
+- Search the [GTN Search](https://training.galaxyproject.org/search) to find prior Q and A, FAQs, Tutorials, and other documentation across all Galaxy resources, to verify in case your issue was already faced by someone.
```

Comment:
```suggestion
- Search the [GTN Search](https://training.galaxyproject.org/search) to find prior Q & A, FAQs, tutorials, and other documentation across all Galaxy resources, to verify in case your issue was already faced by someone.
```